Update 2 after workshop with Bastien

dtonda opened the pull request:
Many changes including :
- Correction of document URL (using gb language)
- Split of databases, emails & collaboration tools
- Adaptation of product names to better align with product catalog

Commit:
Product List Update
Following first review with Bastien - Mostly changes in URLs

Commit:
Multiple Changes after second workshop with Bastien.

yomovh commented:
2 comments (for now):
- Dedicated Servers => I would remove the "s" to Dedicated Server to be consistent with the other 
- NAS-HA : I am suprised to have what seems to be a non english wording : usually we say "High Availability Network Attached Storage" in english no ?

yomovh commented:
Also `Additionnal IP` is not English use `Additional IP` instead

dtonda commented:
> Also `Additionnal IP` is not English use `Additional IP` instead

Changed for next update. Thanks.

> Dedicated Servers

It's spelled like that on the website so for consistency sake, I will keep it as is  : https://www.ovhcloud.com/en-ie/bare-metal/

> NAS-HA : I am suprised to have what seems to be a non english wording : usually we say "High Availability Network Attached Storage" in english no ?

Changed for next update. Thanks. You are right ! The product URL is "nas-ha" but the product name on the page is indeed "HA-NAS"
https://www.ovhcloud.com/en/storage-solutions/nas-ha/
